subgraphs/euler-finance/src/mappings/helpers.ts

```diff
@@ -394,90 +402,118 @@ export function syncWithEulerGeneralView(
+    //update TVL, borrow balance, and deposit balance
+    protocol.totalValueLockedUSD = protocol.totalValueLockedUSD
+      .minus(prevMarketTotalValueLockedUSD)
+      .plus(market.totalValueLockedUSD);
+    protocol.totalBorrowBalanceUSD = protocol.totalBorrowBalanceUSD
+      .minus(prevMarketTotalBorrowBalanceUSD)
+      .plus(market.totalBorrowBalanceUSD);
+    protocol.totalDepositBalanceUSD = protocol.totalDepositBalanceUSD
+      .minus(prevMarketTotalDepositBalanceUSD)
+      .plus(market.totalDepositBalanceUSD);
+
+    log.info(
+      "[syncWithEulerGeneralView]blk={},market={},protocl tvl={},protocol borrowBal={}, protocol DepositBal={}",
+      [
+        event.transaction.hash.toHexString(),
+        market.id,
+        protocol.totalValueLockedUSD.toString(),
+        protocol.totalBorrowBalanceUSD.toString(),
+        protocol.totalDepositBalanceUSD.toString(),
+      ],
+    );
 
-    protocol.totalValueLockedUSD = protocol.totalValueLockedUSD.plus(market.totalValueLockedUSD);
-    protocol.totalBorrowBalanceUSD = protocol.totalBorrowBalanceUSD.plus(market.totalBorrowBalanceUSD);
-    protocol.totalDepositBalanceUSD = protocol.totalDepositBalanceUSD.plus(market.totalDepositBalanceUSD);
-    
     marketUtility.lastUpdateTimestamp = block.timestamp;
     marketUtility.market = market.id;
     marketUtility.twap = eulerViewMarket.twap;
     marketUtility.twapPeriod = eulerViewMarket.twapPeriod;
     marketUtility.save();
-    
+
     updateMarketDailyMetrics(block, market.id, BIGDECIMAL_ZERO);
     updateMarketHourlyMetrics(block, market.id, BIGDECIMAL_ZERO);
   }
   protocol.save();
+  updateFinancials(event.block, BIGDECIMAL_ZERO, "NA");
 }
 
-function updateSnapshotRevenues(
-  marketId: string, 
+export function updateSnapshotRevenues(
+  marketId: string,
   block: ethereum.Block,
-  supplySideRevenueDelta: BigDecimal, 
-  protocolSideRevenueDelta: BigDecimal, 
-  totalRevenueDelta: BigDecimal
+  supplySideRevenueDelta: BigDecimal,
+  protocolSideRevenueDelta: BigDecimal,
+  totalRevenueDelta: BigDecimal,
 ): void {
   let marketDailySnapshot = getOrCreateMarketDailySnapshot(block, marketId);
   let marketHourlySnapshot = getOrCreateMarketHourlySnapshot(block, marketId);
   let financialSnapshot = getOrCreateFinancials(block.timestamp, block.number);
 
   // update daily snapshot
-  marketDailySnapshot.dailySupplySideRevenueUSD = marketDailySnapshot.dailySupplySideRevenueUSD
-    .plus(supplySideRevenueDelta);
-  marketDailySnapshot.dailyProtocolSideRevenueUSD = marketDailySnapshot.dailyProtocolSideRevenueUSD
-    .plus(protocolSideRevenueDelta);
-  marketDailySnapshot.dailyTotalRevenueUSD = marketDailySnapshot.dailyTotalRevenueUSD
-    .plus(totalRevenueDelta);
+  marketDailySnapshot.dailySupplySideRevenueUSD =
+    marketDailySnapshot.dailySupplySideRevenueUSD.plus(supplySideRevenueDelta);
```

Comment:
Is there an extra line in this assignment?

Reply:
what do you mean?

Reply:
nvm, it looked weird in the Review viewer